include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -123,24 +136,32 @@ struct IntegratedActionDataRK4Tpl : public ActionDataAbstractTpl<_Scalar> {
   virtual ~IntegratedActionDataRK4Tpl() {}
 
   VectorXs dx;
+  std::vector<VectorXs> u_diff;
```

Comment:
I've renamed `u_diff` as `ws`, but I prefer to maintain its size to 4 (instead of 3 as you suggest) to have a cleaner code in the RK4 integration loop. Otherwise I would have a shift in the indexes which would be hard to understand. The overhead in terms of memory is minimal, so I think it's worth it.

Reply:
Ok, it seems a reasonable design choice.